alpha/alpha-server/pom.xml

```diff
@@ -79,6 +84,28 @@
       <groupId>org.apache.servicecomb.pack</groupId>
       <artifactId>alpha-spring-cloud-starter-eureka</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.servicecomb.pack</groupId>
+      <artifactId>alpha-spring-cloud-starter-consul</artifactId>
```

Comment:
Not sure if the eureka and consul starter can live together.

Reply:
The default two are disabled and can be enabled separately by parameters.